platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/extensions/handler/AbstractAsyncHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.api.extensions.handler;
+
+import com.swirlds.config.api.Configuration;
+import com.swirlds.logging.api.extensions.event.LogEvent;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public abstract class AbstractAsyncHandler extends AbstractLogHandler {
```

Comment:
I ask myself if we should remove that class for now

Reply:
We definitely should. 
Addressing this in a different PR and marking this as resolved to merge this one.